bigbone/src/test/kotlin/social/bigbone/MastodonClientTest.kt

```diff
@@ -0,0 +1,81 @@
+package social.bigbone
+
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.spyk
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.Response
+import okhttp3.ResponseBody
+import okhttp3.ResponseBody.Companion.toResponseBody
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import social.bigbone.api.exception.BigboneRequestException
+
+class MastodonClientTest {
+
+    private val invalidResponseBody = "{ \"foo\": \"bar\" }"
+    private val v2InstanceResponseBody = "{ \"version\": \"4.0.0rc1\" }"
+    private val v1InstanceResponseBody = "{ \"version\": \"4.0.0rc1\" }"
+
+    @Test
+    fun `should return version of v2 endpoint if available`() {
```

Comment:
I think naming the tests like this helps understanding better what the tests do. Also, using `given/when/then` comments in the code help to separate test setup, execution and assertion statements.